docs/running.md

```diff
@@ -9,14 +9,224 @@ a file `$XDG_STATE_HOME/warnet/warnet.log`, otherwise it will use `$HOME/.warnet
 
 ## Kubernetes
 
-Deploy the resources in `src/templates/`, this sets up the correct permissions on the cluster (`rbac-config.yaml`) and deploys the warnet RPC server as a service + statefulset.
+Deploy the resources in `src/templates/`, this sets up a Kubernetes cluster, apply correct permissions on the cluster (`rbac-config.yaml`), and deploys the warnet RPC server as a service + statefulset.
```

Comment:
I think we can get rid of this information. New users will be distracted by "src/templates", and I think advanced users will discover this information by looking at our collapsed sections.